modules/rtm/R/invert.auto.R

```diff
@@ -73,14 +99,16 @@ invert.auto <- function(observed, invert.options,
       if (!is.numeric(parallel.cores) | parallel.cores %% 1 != 0) {
         stop("Invalid argument to 'parallel.cores'. Must be integer or NULL")
```

Comment:
Hah, good point, I forgot about that. I guess my thinking was that passing too many cores means the user gets the idea but doesn't know how many cores the computer has, while passing a non-integer number of cores means they don't understand how cores work.